apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -235,7 +239,13 @@ public Flux<RifRecordLoadResult> processAsync(
               // Mark active record as complete so progress can be updated.
               .doOnNext(result -> progressTracker.recordComplete(result.getRecordNumber()))
               // Update progress with final result when all records have been processed
-              .doOnComplete(() -> progressTracker.writeProgress())
+              .doOnComplete(
+                  () -> {
+                    progressTracker.writeProgress();
+                    if (fileType == RifFileType.BENEFICIARY) {
```

Comment:
@aschey-forpeople sorry if i missed this but we're not adding a feature flag here?

Reply:
Nope, we decided it's not worth the effort since this is a purely additive change that doesn't affect existing behavior. I forgot to update the ticket though; I'll do that now.

Reply:
We'll still feature flag the server changes, just not the pipeline.

Reply:
kk thanks for the reminder! LGTM then